security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -1283,7 +1283,8 @@ public Builder config(Config config) {
             config.get("header-token").as(TokenHandler.class).ifPresent(this::headerTokenHandler);
             // encryption of cookies
             config.get("cookie-encryption-enabled").asBoolean().ifPresent(this::cookieEncryptionEnabled);
-            config.get("cookie-encryption-password").as(char[].class).ifPresent(this::cookieEncryptionPassword);
+            config.get("cookie-encryption-password").as(Builder::toCookieEncryptionPassword)
```

Comment:
This would be easier:
```java
config.get("cookie-encryption-password").as(String.class)
                    .map(String::toCharArray)
                    .ifPresent(this::cookieEncryptionPassword);
```

Reply:
Yep, thought about this initially but also was thinking that empty string is not allowed hence why I created the custom method. But as you mentioned below, empty password is valid so now I have the opportunity to simplify it this way.